django_ipam/static/django-ipam/js/ip-request.js

```diff
@@ -0,0 +1,34 @@
+/*jslint browser: true*/
+/*global $, jQuery, alert*/
+(function ($) {
+    'use strict';
+    function getAvailableIp() {
+        var e = document.getElementById("id_subnet");
+        var subnet = e.options[e.selectedIndex].value;
+        $.ajax({
+            type: 'GET',
+            url: window.location.origin + "/api/v1/subnet/" + subnet + "/get-first-available-ip",
```

Comment:
you have to get this URL from the admin view context, eg:

- call `reverse()` and add it to the context in the admin class for both `add_view` and `change_view`
- extend the template and add something like `<script>django.ipamGetFirstAvailableIpUrl = '{{ get_first_available_ip_url }}';</script>`
- this line becomes `url: django.ipamGetFirstAvailableIpUrl`